### PR TITLE
Add more info to Schaapi Page

### PR DESCRIPTION
**Pull request by gandreadis**
Also adds meta-titles to all 5 pages.

### CODE REVIEW

File: src/pages/schaapi.js [hunk 1]
```diff
@@ -1,27 +1,102 @@
...
+                    users before a new version of the library is deployed. More specifically, you want to gain insight
+                    into <strong>how users use the library</strong>, and want to see if and{" "}
+                    <strong>how changes affect users</strong>. This will allow you to determine whether the new version
+                    of the library is <strong>backwards compatible</strong>. Finally, after deploying the breaking
+                    changes, you want to notify the affected users of the changes and of a solution to the issue.
+                </p>
+            </div>
+
+            <div className="content-section bg-grey-light">
+                <h2>Schaapi's Approach</h2>
```
Comment (fwdekker) [inline src/pages/schaapi.js]: The other titles are lowercase as well, so I think we should do "Schaapi's approach".

File: src/pages/schaapi.js [hunk 2]
```diff
@@ -1,27 +1,102 @@
...
-        <div className="flex justify-center align-bottom items-center mt-8">
-            <a href="http://swerl.tudelft.nl/bin/view/Main/WebHome">
-                <img src={sergLogo} title="Software Engineering Research Group" height={35} className="mr-4 mt-3" />
-            </a>
-            <a href="https://www.tudelft.nl/">
-                <img src={tudelftLogo} title="Delft University of Technology" height={50} />
-            </a>
+        <div className="mt-16">
+            <div className="content-section bg-delft-blue text-white">
+                <h2>What is a "schaapi"?</h2>
```
Comment (fwdekker) [inline src/pages/schaapi.js]: I don't endorse the term "a schaapi"

Comment (gandreadis) [reply src/pages/schaapi.js]: Then suggest a better title ;)

Comment (gandreadis) [reply src/pages/schaapi.js]: As discussed IRL, changed to "What is Schaapi?"

**Review by fwdekker — APPROVED** [review]
LGTM

File: src/pages/schaapi.js (hunk 2 above)
Comment (fwdekker) [reply src/pages/schaapi.js]: Noice

**Pull request merged**